cirrus/server/tests/conftest.py

```diff
@@ -51,3 +53,101 @@ def fml():
 @fixture
 def exception():
     return Exception("some error")
+
+
+@fixture(name="reset_glean", scope="function", autouse=True)
+def fixture_reset_glean():
+    testing.reset_glean(application_id="cirrus-test", application_version="0.1.0")
+
+
+@fixture
+def get_telemetry():
+    return initialize_glean()
+
+
+@fixture
+def create_recipe():
+    def _create(
+        slug="test",
+        app_id="org.mozilla.test",
+        app_name="test_app",
+        channel="release",
+        feature="test-feature",
+        is_rollout=False,
+        targeting="true",
+        bucket_count=10000,
+        bucket_start=0,
+        bucket_total=10000,
+        bucket_randomization_unit="user_id",
+        is_enrollment_paused=False,
+    ):
+        recipe = {
+            "slug": slug,
+            "appId": app_id,
+            "appName": app_name,
+            "channel": channel,
+            "endDate": None,
+            "locales": None,
+            "branches": [
+                {
+                    "slug": "control",
+                    "ratio": 1,
+                    "feature": {
+                        "value": {"enabled": True},
+                        "featureId": feature,
+                    },
```

Comment:
Cirrus should never get recipes that contain `feature` from Experimenter, so we can remove this bit. It should always get multi-feature recipes without a backup feature tombstone.